### DISCUSSION TITLE
Statamic CP for custom Laravel models

### DISCUSSION THREAD

**Discussion by Glenf** [statamic/cms#7169] [Q&A] [answered]
Hi

Short background information. I maintain a site which has a database that originates from a separate desktop app (still in use). The site was migrated to Laravel about 10 years ago and to Statamic one year ago, users and site conten at first. Now I'm slowly trying to migrate the remaining custom models so that those could be at least viewed in Statamic CP.

I do have this functionality already done with custom Laravel pages but I'm hoping to move it under one UI.

I'd like to create a custom route and page that would look similar as Users CP. Listing, searching, viewing. I tried to understand via docs how this could be achieved. I'm still unsuccesfull with this. 

Can this be done somehow with Collections that would use custom models?

**Comment by duncanmcclean** [comment, marked as answer]
Have you had a look at [Runway](https://statamic.com/addons/duncanmcclean/runway)?

**Reply by Glenf** [reply]
As soon as I posted this, I discovered runway. Thank you, I'll look into it. If this solves my issue I'll mark this as answered.

**Reply by duncanmcclean** [reply]
No worries - hopefully it should do exactly what you're looking for!

**Reply by Glenf** [reply]
Not sure if this is exactly what I'm looking for but it's in the ballpark. I'll need to play around with this for a while. Thanks.
